installing.adoc

```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
 4. `Use {product-short}'s package repository [Y]` +
 displays, press `enter` to continue. Your system will be configured to add
 the {product-short} packaging repository for *yum* or *apt-get* as appropriate
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+{product-short} IMAPD is meant to be installed on a multi-server environment, with
+hundred of thousands of users, and it's still a Beta Release. {product-short} recommends
+to only install it for demo and evaluation purposes.*
+* If installing *{product-short} Network Edition*, you will have the modules Archiving
+and Convertd as an option to add to your {product-short} installation.
+* {product-short}-dnscache is recommended to be installed on the MTA Servers only when
```

Comment:
It is recommended to only install *{product-short}-dnscache* on the MTA Servers  when there are no other DNS servers (for example *dnsmasq* or *bind*) already installed and running on the same server.

Reply:
done